node/src/accountant/scanners/scanners_utils.rs

```diff
@@ -121,6 +119,7 @@ pub mod payable_scanner_utils {
         batch_request_responses: &'a [ProcessedPayableFallible],
         logger: &'b Logger,
     ) -> (Vec<&'a PendingPayable>, Option<Vec<H256>>) {
+        //todo("maybe we can return not tuple but struct with remote_errors_opt member");
```

Comment:
// TODO maybe we can return not tuple but struct with remote_errors_opt member 

Yours looks more like you forgot to drive out a real todo with a panic.